docs/hub/storage-limits.md

```diff
@@ -138,6 +138,8 @@ The super-squash operation compresses your entire Git history into a single comm
 
 ⚠️ **Important**: This is a destructive operation that cannot be undone, commit history will be permanently lost and **LFS file history will be removed**
 
+For datasets with Datasets Viewer enabled, you can also free up storage by squashing Datasets Viewer special refs (`refs/convert/duckdb`, `refs/convert/parquet`). These refs are used by the Datasets Viewer and can accumulate storage over time.
```

Comment:
```suggestion
For datasets with [Datasets Viewer enabled](link to doc about it), you can also free up storage by squashing Datasets Viewer special refs (`refs/convert/duckdb`, `refs/convert/parquet`). These refs are used by the Datasets Viewer and can accumulate storage over time.
```

Reply:
I guess this page https://huggingface.co/docs/hub/en/datasets-viewer-configure

Reply:
yes, looks good